inc/profile.class.php

```diff
@@ -940,7 +940,10 @@ function showFormTools($openform=true, $closeform=true) {
                             'field'      => 'project'),
                       array('itemtype'   => 'ProjectTask',
                             'label'      => _n('Task', 'Task', Session::getPluralNumber()),
-                            'field'      => 'projecttask'));
+                            'field'      => 'projecttask'),
+                      array('itemtype'   => 'ProjectTaskTemplate',
```

Comment:
So, I remove this right at this time. and you will see after for use the same behavior for all objects ?